include/triton/Conversion/TritonGPUToLLVM/TargetInfoBase.h

```diff
@@ -74,6 +74,15 @@ class TargetInfoBase {
                           StringRef message, StringRef file, StringRef func,
                           int line) const = 0;
 
+  // Whether to enable linear layout. This is a per-backend temporary escape
+  // hatch to disable linear layout while figuring out issues. Eventually we
+  // want to enable linear layout everywhere and delete this control.
+  virtual bool enableLinearLayout() const { return true; }
```

Comment:
Hm, I don't see where you use this function.  Maybe this is just a bad merge?

Reply:
Ah, actually I didn't define this function. Seems like a bad merge. I thought you defined the function... Let me try to remove. If that works, then all your previous questions are resolved